src/Bicep.Core/Parser/Lexer.cs

```diff
@@ -621,11 +619,9 @@ private TokenType ScanToken()
             }
         }
 
-        // TODO: Need IsIdStart and IsIdContinue (to disallow starting identifiers with 0-9, for example)
-
-        private static bool IsAlpha(char c) => (c >= 'a' && c <= 'z') || (c >= 'A' && c <= 'Z') || c == '_';
+        private static bool IsIdentifierStart(char c) => (c >= 'a' && c <= 'z') || (c >= 'A' && c <= 'Z') || c == '_';
```

Comment:
The rename was based on feedback from @bterlson long time ago. No actual change was made to the function.

Reply:
Makes sense.